src/test/java/com/saebyeok/saebyeok/domain/MemberTest.java

```diff
@@ -0,0 +1,40 @@
+package com.saebyeok.saebyeok.domain;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class MemberTest {
+
+    private Member member;
+
+    @BeforeEach
+    void setup(){
+        member = new Member(1L, "123456789", "naver", LocalDateTime.now(), false, Role.USER, new ArrayList<>());
+    }
+
+    @Test
+    @DisplayName("사용자의 게시글을 추가하면 리스트에 추가되어야 한다")
+    void addArticleTest() {
+        Article article = new Article(1L, "내용", member, LocalDateTime.now(), true, false, null, new ArrayList<>());
+        member.addArticle(article);
+
+        assertThat(member.getArticles().size()).isEqualTo(1);
```

Comment:
```suggestion
        assertThat(member.getArticles()).hasSize(1);
```
ㅋㅋ 굉장히 사소한거지만...ㅎㅎ
지적할 만한 게 없으니까 이런거라도 지적하게 되네요